src/App.js

```diff
@@ -1,7 +1,46 @@
+import { BrowserRouter, NavLink, Switch, Route } from 'react-router-dom';
 import './App.css';
+import Books from './views/Books/books';
+import Characters from './views/Characters/characters';
+import Films from './views/Films/films';
+import Main from './views/Main/main';
 
 function App() {
-  return <div className="App"></div>;
+  return <div className="App">
+    <BrowserRouter>
+      <header>
+        <NavLink to="/" data-testid="/">
+        Home
+        </NavLink>
+        <NavLink to="/films" data-testid="film-link">
```

Comment:
The "data-testid" is not necessary for these NavLink's as they are not being tested https://testing-library.com/docs/queries/bytestid/